modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommandsStacker.java

```diff
@@ -212,8 +212,7 @@ protected IndexingCommands getOrCreateCommands(DocumentModel doc) {
     }
 
     protected String getDocKey(DocumentModel doc) {
-        // Don't merge commands with different session, so we work only on attached doc
-        return doc.getId() + "#" + doc.getSessionId();
+        return doc.getId();
```

Comment:
I'm wondering, won't it be an issue? cc @bdelbosc

Reply:
I already discussed it with @bdelbosc and we agreed that given that indexing is only done for a single transaction at a time and there's a cleanup of the map for which this is used as the key at the end of the transaction. So not an issue.